src/components/variable.jsx

```diff
@@ -107,7 +109,7 @@ const Variable = React.createClass({
                 {startDate && stopDate &&
                   <h3>Du <FormattedDate value={startDate} /> au <FormattedDate value={stopDate} />&nbsp;:</h3>
                 }
-                <pre><code>{formulas[date].content}</code></pre>
+                <code><SyntaxHighlighter language='python' style={tomorrow}>{formulas[date].content}</SyntaxHighlighter></code>
```

Comment:
@sandcha : I don't know what you mean by JSX content, is there an example we could test it on ?